.github/workflows/dispatch.yaml

```diff
@@ -0,0 +1,44 @@
+name: Dispatch
+
+on:
+    push:
+        branches:
+            - master
+    pull_request:
```

Comment:
Do we need this for pull requests too?

Reply:
Wait, Im just testing 😅 

My idea was to dispatch on pull request and `--apply` on pushes. So you could see what will be dispatched, right on the pull request.

Reply:
I would like to also add a repository_dispatch (not sure if that's the name) so we can run it right in the interface of the actions.

And of course we need the cron , so it runs daily.

Reply:
What I am not sure if we need the push (it could cause a lot of PRs on pushes here, for example if 2 Prs get merged one after another.

Reply:
Let's continue this on the new PR.